qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -1752,19 +1752,19 @@ void FitPropertyBrowser::populateWorkspaceNames() {
     bool allAreAllowed = m_allowedSpectra.isEmpty();
     QStringList tmp;
     auto sv = Mantid::API::AnalysisDataService::Instance().getObjectNames();
-    for (auto &it : sv) {
+    for (const auto &it : sv) {
```

Comment:
Can I ask if we can improve these variable names a bit (things like `tmp`, `it`, `i`, `sv`)? I realise they weren't originally named by you, but it would make this function far more readable